src/__tests__/hoc.node.js

```diff
@@ -13,6 +13,7 @@ tape('hoc', async t => {
   function TestComponent(props) {
     didRender = true;
     t.deepLooseEqual(props.test, testProvides);
+    t.notok(props.ctx, 'does not pass ctx through by default');
```

Comment:
I read this as "no tok" and was confused.

Reply:
lol